appveyor.yml

```diff
@@ -1,41 +1,75 @@
 version: 1.0.{build}
 
-image: Visual Studio 2017
+image:
+  - Visual Studio 2017
+  - Ubuntu
+
 platform: x64
 configuration: Release
 
-clone_folder: c:\projects\DirectXShaderCompiler
-
 environment:
+  APPVEYOR_YML_DISABLE_PS_LINUX: true
   HLSL_SRC_DIR: c:\projects\DirectXShaderCompiler
   HLSL_BLD_DIR: c:\projects\DirectXShaderCompiler\build
-  ARTIFACTS_ZIP_NAME: dxc-artifacts.zip
+  ARTIFACTS_DIR_NAME: dxc-artifacts
+  ARTIFACTS_ZIP_NAME_WINDOWS: $(ARTIFACTS_DIR_NAME).zip
+  ARTIFACTS_ZIP_NAME_LINUX:   $(ARTIFACTS_DIR_NAME).tar.gz
+
+clone_folder: c:\projects\DirectXShaderCompiler
+
+artifacts:
+- path: build\$(configuration)\$(ARTIFACTS_ZIP_NAME_WINDOWS)
+
+# The clone folder and artifacts path is different for the Ubuntu image.
+for:
+-
+  matrix:
+    only:
+      - image: Ubuntu
+  clone_folder: /home/appveyor/projects/DirectXShaderCompiler
+  artifacts:
+  - path: build/$(ARTIFACTS_ZIP_NAME_LINUX)
 
 install:
-- cmd: git submodule update --init
+- git submodule update --init
```

Comment:
guess we didn't really need all the cmd: prefixes before. Oh well makes this change smaller.

Reply:
Yeah, if there's no `cmd:` it runs on both Linux and Windows.